pkg/roachprod/opentelemetry/files/opentelemetry-config.yaml.tmpl

```diff
@@ -0,0 +1,122 @@
+---
+extensions:
+  health_check:
+
+receivers:
+  {{- if .CockroachDBMetrics }}
```

Comment:
Do we ever want to run otelcol and not send crdb metrics?

Reply:
Oh, this is a map? Would it ever be empty?

Reply:
It shouldn't ever be empty but I think your suggestion to always have this receiver configured is a good one. I'll update.